modules/admin_manual/pages/enterprise/firewall/file_firewall.adoc

```diff
@@ -83,7 +83,7 @@ The full list of conditions is (File mimetype upload):
 * ends with
 * doesn’t end with
 
-NOTE: {supported-mimetype-list-url}[The complete list of available mimetypes] that ownCloud supports is available the source of ownCloud core.
+NOTE: {supported-mimetype-list-url}[The complete list of available mimetypes] which ownCloud supports is available in the ownCloud core source.
```

Comment:
As commented in #2319, imho it looks better to have the link on onwcloud/core instead at the beginning. But if you think that my pov is ugly we can keep your PR as it is

Reply:
I think that you may have a good point. I'll play with it for 5 mins.